components/var_dumper/introduction.rst

```diff
@@ -73,6 +73,40 @@ current PHP SAPI:
     #. From time to time, run ``composer global update`` to have the latest
        bug fixes.
 
+Using the VarDumper Component in Your PHPUnit Test Suite
```

Comment:
@nicolas-grekas your should be lowercased. Sorry for that. @WouterJ convinced me in the meantime of how to write this word. :)